Swiftfin tvOS/Views/ItemView/Components/EpisodeSelector/Components/EpisodeCard.swift

```diff
@@ -37,7 +37,7 @@ extension SeriesEpisodeSelector {
                     Image(systemName: "play.fill")
                         .resizable()
                         .frame(width: 50, height: 50)
-                        .foregroundColor(.white)
```

Comment:
We will actually always be in dark mode, so passing in a hard color isn't bad. Additionally, if changing instances of `foregroundColor` we should switch to `foregroundStyle`.